schema-guide.md

```diff
@@ -1776,1070 +2231,1287 @@ An entity can represent different types of entities, such as a team, an institut
+licenses, it is assumed their relationship is OR, not AND.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      license: ...
+      license: Apache-2.0
       type: generic
     ```
     ```yaml
     references:
-      - license: ...
+      - license: Apache-2.0
+        type: generic
+    ```
+    ```yaml
+    preferred-citation:
+      license: 
+        - Apache-2.0
+        - MIT
+      type: generic
+    ```
+    ```yaml
+    references:
+      - license: 
+          - Apache-2.0
+          - MIT
         type: generic
     ```
 
 ### `definitions.reference.license-url`
 
-- **type**: `...`
+- **type**: [`definitions.url`](#definitionsurl)
 - **required**: `false`
-- **description**: ...
+- **description**: The URL of the license text under which the work is licensed (only for non-standard licenses not included in the SPDX License List).
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      license-url: ...
+      license-url: "https://obscure-licenses.com?id=1234"
       type: generic
     ```
     ```yaml
     references:
-      - license-url: ...
+      - license-url: "https://obscure-licenses.com?id=1234"
         type: generic
     ```
 
 ### `definitions.reference.loc-end`
 
-- **type**: `...`
+- **type**: `integer`
 - **required**: `false`
-- **description**: ...
+- **description**: The line of code in the file where the work ends.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      loc-end: ...
+      loc-end: 42
       type: generic
     ```
     ```yaml
     references:
-      - loc-end: ...
+      - loc-end: 42
         type: generic
     ```
 
 ### `definitions.reference.loc-start`
 
-- **type**: `...`
+- **type**: `integer`
 - **required**: `false`
-- **description**: ...
+- **description**: The line of code in the file where the work starts.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      loc-start: ...
+      loc-start: 21
       type: generic
     ```
     ```yaml
     references:
-      - loc-start: ...
+      - loc-start: 21
         type: generic
     ```
 
 ### `definitions.reference.location`
 
-- **type**: `...`
+- **type**: [`definitions.entity`](#definitionsentity)
 - **required**: `false`
-- **description**: ...
+- **description**: The location of the work.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      location: ...
+      location:
+        name: "Library of the Unseen University"
       type: generic
     ```
     ```yaml
     references:
-      - location: ...
+      - location:
+          name: "Library of the Unseen University"
         type: generic
     ```
 
 ### `definitions.reference.medium`
 
-- **type**: `...`
+- **type**: Nonempty `string`
 - **required**: `false`
-- **description**: ...
+- **description**: The medium of the work.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      medium: ...
+      medium: "datasette"
```

Comment:
bit confusing maybe, perhaps we could use something like `Betamax` for humoristic purposes?

Reply:
I see (also: includes a typo which makes `dataset` a substring, not good).
Using "5¼-inch floppy disk" now :).